vision/product_search/add_product_to_product_set_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package productsearch
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestAddProductToProductSet(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	const location = "us-west1"
+	const productSetDisplayName = "fake_product_set_display_name_for_testing"
+	const productSetID = "fake_product_set_id_for_testing"
+	const productDisplayName = "fake_product_display_name_for_testing"
+	const productCategory = "homegoods"
+	const productID = "fake_product_id_for_testing"
+
+	var buf bytes.Buffer
+
+	// Create fake product set and product.
+	if err := createProductSet(&buf, tc.ProjectID, location, productSetID, productSetDisplayName); err != nil {
+		t.Fatal(err)
+	}
+	if err := createProduct(&buf, tc.ProjectID, location, productID, productDisplayName, productCategory); err != nil {
+		t.Fatal(err)
+	}
+
+	// Make sure the product is not in the product set.
+	buf.Reset()
+	if err := listProductsInProductSet(&buf, tc.ProjectID, location, productSetID); err != nil {
+		t.Fatal(err)
+	}
+	if got := buf.String(); strings.Contains(got, productID) {
+		t.Errorf("Product ID %s already in product set", productID)
+	}
+
+	// Add product to product set
```

Comment:
Not sure we need this comment. If you want to keep it (totally OK), please add a period at the end.